MAUI/DataGrid/Column-sizing.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: Column Sizing in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Column Sizing support in Syncfusion MAUI DataGrid (SfDataGrid) control and more.
+platform: maui
+control: SfDataGrid
+documentation: ug
+---
+
+# Column Sizing in .NET MAUI DataGrid (SfDataGrid)
+
+DataGrid allows you to set the column widths based on certain logic using `SfDataGrid.ColumnWidthMode` or `DataGridColumn.ColumnWidthMode`property. Below is the list of predefined column sizing options available.
```

Comment:
The DataGrid allows you to set the column widths based on certain logics using the `SfDataGrid.ColumnWidthMode` or `DataGridColumn.ColumnWidthMode`property. Below is the list of predefined column sizing options available: